app/scenes/Document/components/EditableTitle.tsx

```diff
@@ -253,7 +257,7 @@ const EditableTitle = React.forwardRef(function _EditableTitle(
           value={document.emoji}
           onChange={handleEmojiChange}
           // Restore focus on title
-          onClickOutside={handleClick}
+          onClickOutside={restoreFocus}
```

Comment:
It's no longer possible to click and move the caret in the title due to this change

Reply:
Nope! It's because of the `extraArea` mixin :see_no_evil: